gnomad_qc/v4/annotations/insilico_predictors.py

```diff
@@ -514,21 +514,21 @@ def get_revel_for_unmatched_transcripts() -> None:
     Create Tables with alternative REVEL scores for variants in v4.1 release.
 
     ..note:
-        REVEL was computed using transcripts from Ensembl v64. However, in gnomAD
```

Comment:
I kept the first paragraph as it was for the reasons in my comments and accepted your suggestions for the rest.